spec/spec_helper.rb

```diff
@@ -83,10 +83,21 @@ def policy_class
   end
 end
 
+class CommentScope
+  attr_reader :original_object
+  def initialize(original_object)
+    @original_object = original_object
+  end
+
+  def ==(other)
+    original_object == other.original_object
+  end
+end
+
 class CommentPolicy < Struct.new(:user, :comment)
   class Scope < Struct.new(:user, :scope)
     def resolve
-      scope
+      CommentScope.new(scope)
```

Comment:
Why did you change this?

Reply:
This spec helper was using the old pattern of returning `scope` directly. I didn't think it made sense to leave that as example code when we're discouraging that pattern's use. I updated it to return a simple mock scope object, so we can test that the policy is correctly wrapping the object it's given.

Reply:
Hmm, I think returning this CommentScope object makes the test more confusing that to just return the `scope`. In the test we just want to know that the `resolve` method returns the expected value, whatever it is.

Can you revert the `CommentScope` code for now and I'll merge the generator fix?

Reply:
Or nah... Let's keep it as it is :) Thanks!